src/components/tilingsystem/edgeTilingManager.ts

```diff
@@ -81,12 +81,14 @@ export default class EdgeTilingManager {
         this._rightCenter.width = this._topRight.width;
     }
 
-    public canActivateEdgeTiling(x: number, y: number) {
+    public canActivateEdgeTiling(
+        pointerPos: { x: number; y: number }
+    ): boolean {
         return (
-            x <= this._workArea.x + EDGE_TILING_OFFSET ||
-            y <= this._workArea.y + TOP_EDGE_TILING_OFFSET ||
-            x >= this._workArea.x + this._workArea.width - EDGE_TILING_OFFSET ||
-            y >= this._workArea.y + this._workArea.height - EDGE_TILING_OFFSET
+            pointer.x <= this._workArea.x + EDGE_TILING_OFFSET ||
```

Comment:
The extension stops working here because pointer is undefined. That's clearly a typo (`pointer` instead of `pointerPos`).
After fixing it, I was able to test your changes.

Reply:
Stupid last minute fixups.... I'll make it right and test it with multiple monitors.